src/service/ui/messaging.js

```diff
@@ -117,7 +117,7 @@ function getShortTime(time) {
 
         case (diff < GLib.TIME_SPAN_HOUR):
             // TRANSLATORS: Time duration in minutes (eg. 15 minutes)
-            return _('%d minutes').format(diff / GLib.TIME_SPAN_MINUTE);
+            return gsconnect.ngettext('%d minute', '%d minutes', (diff / GLib.TIME_SPAN_MINUTE)).format(diff / GLib.TIME_SPAN_MINUTE);
```

Comment:
This is a weird
```suggestion
            return ngettext('%d minute', '%d minutes', (diff / GLib.TIME_SPAN_MINUTE)).format(diff / GLib.TIME_SPAN_MINUTE);
```
This is a non-obvious situation, but when under the "GSConnect process" the gettext functions are directly on the `window` (global) variable. When under the gnome-shell process they're on the `window.gsconnect` variable because otherwise we would clobber gnome-shell's (since they already have _their's_ on the global object).

Reply:
Aha! Well, that's the reason I submitted this as a pull request instead of directly committing it. 😃

(Actually, the reason is mostly that I didn't actually _run_ these changes, I only tested the `.pot` file generation. I'm still squatting on v12 waiting for v14 to get reviewed, and feeling more stupid every day that doesn't happen.)

Reply:
Yeah, that's why I decided might as well get v15 on the go, too :P